packages/botonic-react/src/webchat/webchat.jsx

```diff
@@ -581,6 +581,7 @@ export const Webchat = forwardRef((props, ref) => {
     updateWebchatSettings: settings => {
       const themeUpdates = normalizeWebchatSettings(settings)
       updateTheme(merge(webchatState.theme, themeUpdates), themeUpdates)
+      updateLastMessageDate(new Date())
```

Comment:
does this depend on #1360? there you're sending new Date().toISOString

Reply:
yes, it is related and it depends on it (but this one was created before). Changing this PR to draft